packages/base-styles/_colors.scss

```diff
@@ -25,6 +25,15 @@ $light-gray-200: #f3f4f5;
 $light-gray-100: #f8f9f9;
 $white: #fff;
 
+// G2 colors.
+$dark-gray-primary: #1e1e1e;
+$medium-gray-text: #757575;
+$light-gray-secondary: #ccc;
+$light-gray-tertiary: #e7e8e9;
+$theme-color: theme(button);
+$blue-medium-focus: #007cba;	// @todo: Currently being used as "spot" color. Needs to be considered in context of themes.
+$blue-medium-focus-dark: #fff;
```

Comment:
that's white, not blue :) I believe ideally the variable name should contain "blue" but be more about the usage.

Reply:
We could rename both of these to $block-focus and $block-focus-dark? Because that's what they are.

Reply:
#fff for dark?  

Is this just about the blocks or all focus styles (buttons,...), I'd rather avoid the "block" prefix and maybe replace with something more generic.

Reply:
This is only for blocks, because we can't really rely on the theme colors to bring sufficient contrast in dark themes. In light themes:

<img width="706" alt="Screenshot 2020-02-20 at 13 16 00" src="https://user-images.githubusercontent.com/1204802/74932956-4b0c4b80-53e3-11ea-870a-feb9250274a7.png">

In dark themes:

<img width="685" alt="Screenshot 2020-02-20 at 13 16 24" src="https://user-images.githubusercontent.com/1204802/74932962-4e9fd280-53e3-11ea-9baa-1450102801e2.png">